king_phisher/client/application.py

```diff
@@ -245,6 +255,12 @@ def campaign_configure(self):
 
 		assistant.interact()
 
+	def do_campaign_created(self, campaign_id):
+		pass
+
+	def do_campaign_deleted(self, campaign_id):
+		pass
```

Comment:
This could be updated to be an actual handler:
```
self.rpc('db/table/delete', 'campaigns', campaign_id)
```
Then the caller would only need to emit the signal.